src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,88 @@
 import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  todos: Todo[];
+  activeTodoId: number;
+  setActiveTodoId: React.Dispatch<React.SetStateAction<number>>;
```

Comment:
Better destructure Dispatch and SetStateAction from react to improve readability